test/TestAlgorithm.cpp

```diff
@@ -203,768 +204,1239 @@ TEST(algorithm, intersectarrays) {
    TEST(X, gpu_test_##Y) { gpu_test_##X##Y(); } \
    static void gpu_test_##X##Y()
 
-GPU_TEST(algorithm, fill_empty) {
```

Comment:
These are intentionally removed as they do not test anything different than the host versions.